packages/kit/src/core/create_app/index.js

```diff
@@ -62,7 +75,10 @@ function generate_client_manifest(manifest_data, base) {
 			.join(',\n\t\t\t\t')}
 	]`.replace(/^\t/gm, '');
 
-	/** @param {string[]} parts */
+	/**
+	 * Matches components by path and returns an array of those components accessed within the `c`array.
```

Comment:
```suggestion
	 * Matches components by path and returns an array of those components accessed within the `c` array.
```

Reply:
`accessed`? I'm not sure what that's trying to communicate, but it doesn't sound right

Reply:
I was trying to communicate that the array returned contains references to the import functions listed in the `c` array within the generated code. I'm unsure of the best way to describe the form of the arrays it returns, since just saying they contain the components seems a little wrong